cypress/support/commands.ts

```diff
@@ -94,12 +96,16 @@ Cypress.Commands.overwrite("visit", (original, url, options) => {
       onBeforeLoad(win) {
         options && options.onBeforeLoad && options.onBeforeLoad(win)
         win.localStorage.clear()
-        const provider = new JsonRpcProvider(
-          Cypress.env("PROVIDER_HOST"),
-          { name: "local", chainId: Cypress.env("NETWORK_ID") },
+        const provider = new JsonRpcProvider(Cypress.env("PROVIDER_HOST"), {
+          name: "local",
+          chainId: Cypress.env("NETWORK_ID"),
+        })
+        const signer = new Wallet(
+          Cypress.env("PRIVATE_TEST_WALLET_PK"),
+          provider,
         )
-        const signer = new Wallet(Cypress.env("PRIVATE_TEST_WALLET_PK"), provider)
         win.ethereum = new CustomizedBridge(signer, provider)
+        win.ethereum.isMetaMask = true
```

Comment:
Ideally we'd only change this property for the test that requires it, but this is fine for now.